providers/postgres/src/airflow/providers/postgres/hooks/postgres.py

```diff
@@ -173,6 +178,24 @@ def get_conn(self) -> connection:
         self.conn = psycopg2.connect(**conn_args)
         return self.conn
 
+    def _get_pandas_df(
```

Comment:
```suggestion
    def get_pandas_df(
```
I think this shouldn't be private

Reply:
Hi, I’ve updated the code based on your feedback. Since the method you pointed out is a private one from the parent DbApiHook, I avoided modifying it directly. Instead, I reimplemented get_df here and used the private method only for the Polars case. Let me know if you’d recommend a different approach. I’ll also add unit tests for this once it’s approved.

@bugraoz93